katas/content/distinguishing_states/four_basis_states/Solution.qs

```diff
@@ -0,0 +1,8 @@
+namespace Kata {
+    operation BasisStateMeasurement(qs : Qubit[]) : Int {
+        // Measurement on the first qubit gives the higher bit of the answer, on the second - the lower
```

Comment:
We can omit the comments in the code itself, since we've just explained the logic above in more detail.
```suggestion
```